storage_file/models/storage_file.py

```diff
@@ -135,20 +135,33 @@ def _compute_data(self):
             else:
                 rec.data = None
 
-    @api.depends("backend_id.served_by", "backend_id.base_url", "relative_path")
+    @api.depends(
+        "backend_id.served_by",
+        "backend_id.base_url",
+        "backend_id.url_include_directory_path",
+        "relative_path",
+    )
     def _compute_url(self):
```

Comment:
@simahawk Are we sure that compute methods are triggered if the values provided by the `server.env.mixin` changes?

Reply:
hmm very good point. @guewen what is the recommended way in this case?
Context: this is a computed + stored field depending on server env params.

Reply:
@simahawk IMO it's not a blocking point. It would probably be necessary to add a warning in the README on this point and propose a procedure to launch this recomputation manually (via click-odoo or ????) if these values are modified.

Reply:
not sure. Can be risky whenever you move a db from prod to integration for instance.
Would be nice to have a common well-known solution.

Reply:
It kind of defeats the purpose of environment, since a copy of a prod database to another environment will not reset the url.

Does it need to be stored?

Reply:
I don't think so. This computation is not so heavy and is not used so often AFAIR.

Reply:
That's the point. I share the same POV. The url should no more be stored...  If I remember well, into the past, the url field was used by a controller to lookup for the record... it's no more the case and it seems safe to no more store this value into the database.